proposals/0095-AOA-multiplexing.md

```diff
@@ -124,53 +128,30 @@ Extends SdlRouterService so that internal class (RegisteredApp) is aware of Tran
 	}
 ```
 
-### SdlRouterService uses (newly added) ITransportWriter to write into actual transport
-Introduces BluetoothTransportWriter and AoaTransportWriter class that utilize actual transports to send data to SDL Core:
-```java
-public interface ITransportWriter {
-	boolean writeBytesToTransport(Bundle bundle);
-	boolean manuallyWriteBytes(byte[] bytes, int offset, int count);
-}
-
-// BluetoothTransportWriter internally utilizes MultiplexBluetoothTransport
-public class BluetoothTransportWriter implements ITransportWriter {
-    private MultiplexBluetoothTransport mSerialService = null;
-    ...
-    public boolean writeBytesToTransport(Bundle bundle){
-        ...
-    }
-    
-    public boolean manuallyWriteBytes(byte[] bytes, int offset, int count){
-        ...
-    }
-}
-
-// AoaTransportWriter internally utilizes (new) MultiplexAOATransport
-public class AoaTransportWriter implements ITransportWriter {
-    private MultiplexAOATransport mSerialService = null;
-
-    public boolean writeBytesToTransport(Bundle bundle){
-        ...
-    }
-    
-    public boolean manuallyWriteBytes(byte[] bytes, int offset, int count){
-        ...
-    }
-}
-```
-
-SdlRouterService selects either BluetoothTransportWriter or AoaTransportWriter according to the TransportType of the RegisteredApp instance:
+### SdlRouterService uses either MultiplexBluetoothTransport or MultiplexAoaTransport depending on the TransportType
 ```java
 public class SdlRouterService extends Service {
     ...
-    // TransportWriters -- for now BT and USB.
-	private BluetoothTransportWriter bluetoothTransportWriter;
-	private AoaTransportWriter aoaTransportWriter;
+    // underlying transport dependning on the TranportType.
```

Comment:
Please update to "underlying transport depending on the TransportType."

Reply:
Hi @theresalech, thanks for the review.
Fixed the typo you pointed out. Thanks.

Reply:
@shiniwat it looks like there is still one typo.  "TranportType" should be updated to "TransportType" (add an "s").